lib/store_events.js

```diff
@@ -8,6 +8,8 @@ async function storeEvents(exporter, events) {
   try {
     if (exporter.exporter_name === 'bsc_receipts_exporter') {
       await exporter.sendDataWithKey(events, 'transactionHash');
+    } else if (exporter.exporter_name === 'utxo-litecoin-v2') {
```

Comment:
We want to have this logic for all UTXO based blockchains, so let's use the `BLOCKCHAIN` env variable as distinguisher.